docs/book/how-to/develop-locally/keep-your-pipeline-dashboard-clean.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Learn how to keep your pipeline runs clean during development.
+---
+
+# Keep your pipelines and dashboard clean
+
+When developing pipelines, it's common to run and debug them multiple times. To avoid cluttering the server with these development runs, ZenML provides several options:
+
```

Comment:
maybe the first option should be to just disconnect from the remote server and run with a default stack on a local zenml installation?

Reply:
maybe worth adding the zenml clean option for local development?

Reply:
Both added.